test/upload-download.test.js

```diff
@@ -91,6 +91,7 @@ describe('storage service', function() {
       .set('Content-Type', 'application/json')
       .expect('Content-Type', /json/)
       .expect(200, function(err, res) {
+        if (err) done(err);
```

Comment:
okay, looked at other places in the code also `returns` the err
i guess it would avoid `done()` calling more than once